src/main/java/frc/robot/OI.java

```diff
@@ -14,4 +22,23 @@ public double getRightY() {
     public double getLeftX() {
         return left.getX();
     }
+
+    public OI() {
+        IntakeRoller intakeRoller = IntakeRoller.getInstance();
+        IntakePlacer intakePlacer = IntakePlacer.getInstance();
+        Button intake = controller.getRBButton();
+        intake.whenPressed(new IntakeCargo());
```

Comment:
`controller.getRBButton().whenPressed(new IntakeCargo()):` ?

Reply:
That just looks worse